src/components/App.js

```diff
@@ -6,9 +6,11 @@ import { GET_ALL_ITEMS } from '../api';
 import { Route, Routes, NavLink } from 'react-router-dom';
 import { useState, useEffect } from 'react';
 import Checkout from './Checkout';
-import '../styles/_App.scss'
 import ShoppingBag from './ShoppingBag';
 import Home from './Home';
+import Success from './Sucess';
```

Comment:
Little typo here in component file name.

Reply:
oh nice catch